packages/desktop-ui-lib/src/lib/permission-manager/permission-manager.component.html

```
@@ -0,0 +1,309 @@
+					</div>
+				</nb-card-body>
+			</nb-card>
+		</div>
+	</div>
+}
+
+<!-- ─── SHARED: Status badge template ───────────────────────────────── -->
+<ng-template #statusBadge let-status="status">
+	@switch (status) {
+		@case ('granted') {
+			<button nbButton status="success" size="tiny" outline>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)